conftest.py

```diff
@@ -6,6 +6,8 @@
 PLATFORM_INFO = {"darwin": "mac", "linux": "linux", "win32": "windows"}
```

Comment:
this file does belong in the test root of `jupyterfs/tests`, not sure if this move was an attempt to fix issues?

Reply:
pytest will strongly insist on the conftest needing to be in project root in order to set `pytest_plugins` key: 
https://docs.pytest.org/en/latest/how-to/writing_plugins.html#requiring-plugins-in-non-root-conftests

Reply:
"which is not located in the tests root directory is deprecated"

The tests root is `jupyterfs/tests`, but maybe we need to remove the [spurious __init__ file](https://github.com/jpmorganchase/jupyter-fs/blob/main/jupyterfs/tests/__init__.py)

Reply:
I agree it would make sense if it was in test, but it seems very hard to get pytest to accept it. Removing the `__init__.py` file just makes relative imports within the test folder stop working, so I don't really have any good solutions. Are there any particular flows that will break for you if it is not included in that folder?

Reply:
building fine without changes on my fork, so I don't think the change is necessary: https://github.com/timkpaine/jupyter-fs/pull/20

Reply:
Noted. It works for me as well as long as I always pass `jupyterfs/tests` on the CLI. C.f. https://docs.pytest.org/en/7.1.x/reference/customize.html#initialization-determining-rootdir-and-configfile A little cumbersome, but I'll survive.